### DISCUSSION TITLE
Failed to execute goal org.gephi:gephi-maven-plugin:1.2.5-SNAPSHOT:validate (validate-plugin) on project gephi-plugins: The 'licenseName' configuration should be set for the project 'Map Plugin'.

### DISCUSSION THREAD

**Discussion by Changgf** [gephi/gephi-plugins#246] [Q&A] [answered]
Failed to execute goal org.gephi:gephi-maven-plugin:1.2.5-SNAPSHOT:validate (validate-plugin) on project gephi-plugins: The 'licenseName' configuration should be set for the project 'Map Plugin'. This can be added to the configuration of the 'nbm-maven-plugin' plugin. In addition, a 'licenseFile' can be specified, relative to the module's root folder. -> [Help 1]

To see the full stack trace of the errors, re-run Maven with the -e switch.
Re-run Maven using the -X switch to enable full debug logging.

For more information about the errors and possible solutions, please read the following articles:
[Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

After correcting the problems, you can resume the build with the command
  mvn <args> -rf :gephi-plugins

**Comment by mbastian** [comment, marked as answer]
Hi, the `licenseName` is indeed a mandatory field for every plugin so it's normal you get a validation error. You can read more details on the [plugin's documentation](https://github.com/gephi/gephi-maven-plugin#mvn-orggephigephi-maven-pluginvalidate).
